src/sidecar/connections/watcher.test.ts

```diff
@@ -55,21 +54,12 @@ describe("sidecar/connections/watcher.ts ConnectionStateWatcher handleConnection
     sandbox.restore();
   });
 
-  it("should parse valid Connection-like JSON objects correctly", async () => {
-    // since websockets don't go through the same client code as the (HTTP) REST API, we need to
-    // test that the watcher can coerce the JSON object into a Connection object correctly
-    const connectionObj = ConnectionToJSONTyped(TEST_AUTHENTICATED_CCLOUD_CONNECTION);
-    assert.equal(
-      // not Date
-      typeof connectionObj["status"]["authentication"]["requires_authentication_at"],
-      "string",
-    );
-
+  it("should basically work when given good message", async () => {
```

Comment:
Preserve good bits of this test, but a non-decoded message will never make it this far anymore.